### DISCUSSION TITLE
2주차 파워앱스 캔버스앱 만들때 siteinspector 화면에 데이터가 없이 나타납니다.

### DISCUSSION THREAD

**Discussion by gwh-21c** [fusiondevkr/pl900#65] [1 Q&A]
![20210902_233407](https://user-images.githubusercontent.com/69518826/131863147-3f09013a-61c2-4bed-bc80-8ffb580f953c.jpg)
![20210902_233407](https://user-images.githubusercontent.com/69518826/131863255-d6c75032-4ad8-4301-b981-2fad743491f1.jpg)
개발자 계정의 onedrive에 Contoso Site Tracking.xlsx를 저장하고 만들기를 했는데 화면이 빈화면이 나오고 데이타는 나오지를 않고 있습니다.
OneDrive에는 사진이나 액셀 파일이 모두 저장되어 있는데요.
무엇을 잘못했는지 모르겠습니다. 
도움 요청 드립니다.

**Comment by justinyoo** [comment]
.zip 파일 풀면 엑셀 파일이랑 이미지 파일 들어있는 폴더가 나오는데요, 둘 다 원드라이브로 옮기셔야 해욥! 현재 이미지 파일 경로를 못 찾아서 그럴 수 있습니다
